ckanext/switzerland/dcat/harvesters.py

```diff
@@ -18,9 +19,20 @@ def info(self):
             'description': 'Harvester for DCAT-AP Switzerland datasets from an RDF graph'  # noqa
         }
 
+    def _set_config(self, config_str):
```

Comment:
Do not invent something new here, override `validate_config` from the super class instead